OrbitLinuxTracing/UprobesUnwindingVisitor.cpp

```diff
@@ -100,6 +134,17 @@ UprobesCallstackManager::ProcessUretprobesCallstack(
   return full_callstack;
 }
 
+void UprobesCallstackManager::ProcessUretprobesWithoutCallstack(pid_t tid) {
```

Comment:
Can the function above (ProcessUretprobesCallstack) be deleted?

Reply:
Done.